<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>pBasex</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="plainText">
      <string>Enter the dimensions of the polar coordinates and the number of radial                                   basis functions</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Azimutal</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_r">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_phi">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Rbasis">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Radial </string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Radial basis</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Polar binning</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_savePES">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Save PES</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_saveANG">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Save ANG</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_savePROC">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Save PROC</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_saveINV">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Save INV</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>File</string>
    </attribute>
    <widget class="QSpinBox" name="spinBox_bitdepth">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Bit depth</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>Please set the parameters for openning file as 2D matrix.</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_width">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_height">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Binary file</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>ASCII file</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_skiplines">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip lines</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_transpose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Transpose</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Multiple files</string>
    </attribute>
    <widget class="QSpinBox" name="spinBox_quadrant">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Quadrant</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>File type</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="radioButton_ASCII">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ASCII</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_binary">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Binary</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_orig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Original VMI ASCII</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_inv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Inverted image ASCII</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PES</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_betas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>betas</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Saving files</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Chose parameters for multiple file inversion</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="plainText">
      <string>&quot;0&quot; - all image is used</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_subtract_bkg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Subtract bkg</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Image display</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>Settings for 2D image display</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Circle:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_thickness">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Line thickness:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Line colour:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_getcolour">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>GetColour</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Tab</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="plainText">
      <string>Nothing is here</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
